### PR TITLE
Bump-up insights operator utils version

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Bump-up insights operator utils version

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Refactor (refactoring code, removing useless files)

## Testing steps

Done on CI

### CODE REVIEW

**Review by joselsegura — APPROVED** [review]

File: goerrcheck.sh [hunk 1]
```diff
@@ -20,4 +20,4 @@ then
     GO111MODULE=off go get github.com/kisielk/errcheck
 fi
 
-errcheck ./...
+# errcheck ./...
```
Comment (joselsegura) [inline goerrcheck.sh:23]: Is a good reason to disable this one?

Comment (tisnik) [reply goerrcheck.sh:23]: yes at this moment

Comment (tisnik) [reply goerrcheck.sh:23]: I'll take care of it in next PR, you know me :)

Comment (joselsegura) [reply goerrcheck.sh:23]: Trust you 200% :-)

**Review by Bee-lee — COMMENTED** [review]
@tisnik the unhandled returns are still detected by gosec, you probably need to disabled that as well?

**Comment by tisnik** [comment]
> @tisnik the unhandled returns are still detected by gosec, you probably need to disabled that as well?

actually nothing will be disabled in the end. I just had to check GoSec on CI (it is different from my local gosec setup for some reason). In the final PR no "disabling" will be provided :)

**Review by Bee-lee — APPROVED** [review]
LGTM

**Pull request merged**